contents/docs/user-guides/support.md

```diff
@@ -0,0 +1,143 @@
+|[Security assessment](#security-assessment)|Partial|Partial|Partial|✔️|
+|[Bespoke pricing](#bespoke-pricing)|x|x|x|✔️|
+|[Payment via invoicing](#payment-via-invoicing)|x|x|Minimum $2k/month spend|✔️|
+|[Downtime developer pairing](#downtime-developer-pairing)|x|x|x|✔️|
+|[Region-specific private cloud infrastructure managed by PostHog](#region-specific-cloud-infrastructure-managed-by-posthog)|x|x|x|✔️|
+
+## Community Support
+
+Whilst Community Support offers no guarantees, there are two places to ask questions if you're getting stuck and want to see if someone in the community could help you:
+
+* We have >1,300 developers in [our public Slack group](../slack).
+* Feel free to open an issue in our [main project's GitHub repo](https://github.com/posthog/posthog).
+
+We are very grateful for folks that do this the other way around and help answer others' questions. You may just end up with some [merch](https://merch.posthog.com/) for particularly good answers :)
+
+## Dedicated Slack
+
+Our preference is to support customers through a private Slack channel. This is because it helps teams realize they're on the same side - it's just more fun for everyone, which helps drive engagement.
+
+From our side we will select the right team for you - typically this would be:
+
+* Customer Success contact (whoever you spoke with ahead of becomign a paying customer)
+* Support hero engineer (we have an on call engineer between 2am PT / 9am GMT and 5pm PT / 12am GMT)
+* Platform team engineers (help with deeper infrastructure or ingestion questions)
+* Product manager (helps us to understand feature requests, or can clarify current feature set)
+
+[These are the kinds of people](/handbook/company/team) you'd be working with.
+
+Notice - we do *not* have non technical support people who triage issues. We directly connect you to people who are building, designing or who sold the deal. This has scaled to support thousands of companies very effectively - we always aim to productize solutions to problems.
+
+From your side, it's important you bring your A team. We recommend you add:
+
+* A stakeholder from engineering
+* A stakeholder from data engineering / data analysis if relevant
+* A stakeholder from product management
+* 2-3 power users
+* A stakeholder from customer success if relevant
+* A stakeholder from sales if relevant
+
+It's best not to let your team get too much bigger as it can create a lot of noise. We don't support for example you adding 125 end users of PostHog to this group - you need to triage your own issues with larger volumes.
+
+## Dedicated email
+
+For Enterprise customers that can't use Slack easily, we can offer priority email support instead. 
+
+## Training sessions
+
+We run a set number of group training sessions with your end users. We will walk them through:
+
+* Understanding product analytics performance
+* Diagnosing why your analytics are performing well or badly
```

Comment:
I feel this is a little over complicated, how about "How to make your product more successful?"

Reply:
Done!